samples/excel/82-document/get-file-in-slices-async.yaml

```diff
@@ -0,0 +1,201 @@
+id: excel-82-document-get-file-in-slices-async-
+name: Get file in slices and recombine
+description: Get current file in slices and recombine the slices afterwards.
+host: EXCEL
+api_set:
+    ExcelApi: 1.1
+script:
+    content: |-
+        $("#setup").click(() => tryCatch(setup));
+        $("#get-file").click(() => tryCatch(getCurrentFile));
+
+
+        function getCurrentFile() {
+            const sliceSize = 4096 /*Bytes*/
+
+            // Note: If you don't specify the slice size, a default will be assumed for you.  In this sample. to demonstrate how the slices work, a small size limit is specified.
+            Office.context.document.getFileAsync(Office.FileType.Compressed, { sliceSize: sliceSize },
+                function (result) {
+                    if (result.status === Office.AsyncResultStatus.Failed) {
+                        return onError(result.error);
+                    }
+
+                    // Result.value is the File object.
+                    getFileContents(result.value, onSuccess, onError);
+                });
+
+            function onError(error: Office.Error): void {
+                OfficeHelpers.UI.notify('Error', error.message);
+                OfficeHelpers.Utilities.log(error);
+            }
+
+            function onSuccess(byteArray: number[]) {
+                // Now that all the file content is stored in the "data" parameter,
+                // you can do something with it, such as print, store into database, etc.
```

Comment:
Suggest that you change the comment to:

// Now that all of the file content is stored in the "data" parameter,
// you can do something with it, such as print the file, store the file in a database, etc.

Reply:
Thanks, done.